### PR TITLE
Add basic task functionality

### LINKED ISSUE #45: As a Teaching Assistant, I want to be able to manage (CRUD) any student’s task list.

### PR DESCRIPTION
**Pull request by gunbux**
Resolves: #45

### CODE REVIEW

File: src/main/java/seedu/address/model/task/UniqueTaskList.java [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+    }
+
+    /**
+     * Replaces the contents of this list with {@code tasks}.
+     * {@code tasks} must not contain duplicate persons.
+     */
+    public void setTasks(List<Task> tasks) {
+        requireAllNonNull(tasks);
+        if (!tasksAreUnique(tasks)) {
+            throw new DuplicateStudentException();
```
Comment (LuYiting0913) [inline src/main/java/seedu/address/model/task/UniqueTaskList.java]: DuplicateTask?

Comment (gunbux) [reply src/main/java/seedu/address/model/task/UniqueTaskList.java]: Yep good catch

**Review by LuYiting0913 — COMMENTED** [review]
lgtm

**Comment by tensaida** [comment]
lgtm

**Pull request merged**